src/test/java/uk/co/ramp/covid/simulation/integrationTests/MovementTest.java

```diff
@@ -445,17 +446,17 @@ public void positiveTestsStayInQuarantine() {
 
         per.infect();
         per.getcVirus().forceSymptomatic(true);
-        // Usually there's a delay before symptonms but we just force it here
-        double time = per.getcVirus().getSymptomDelay() + 1;
-        for (int j = 0; j < time; j++) {
+
+        //  Usually there's a delay before symptonms but we just force it here
+        while (!per.getcVirus().isSymptomatic()) {
```

Comment:
This looks dangerous.  If something goes wrong, this test could get stuck in an infinite loop.

Reply:
I agree - I'm planning on adding some features (probably in another PR) that gets the number of steps we need to advance from the COVID object itself.

There's a couple of these types of loops that have bitten in the past.